pkgs/tools/security/sequoia/default.nix

```diff
@@ -1,32 +1,47 @@
-{ stdenv, fetchFromGitLab, lib, darwin
-, git, nettle, llvmPackages, cargo, rustc
-, rustPlatform, pkgconfig, glib
-, openssl, sqlite, capnproto
-, ensureNewerSourcesForZipFilesHook, pythonSupport ? true, pythonPackages ? null
+{ stdenv
+, fetchgit
+, lib
+, darwin
+, git
+, nettle
+# Usually we'd like this to be the same version as used by rust
+, llvmPackages_10
+, cargo
+, rustc
+, rustPlatform
+, pkg-config
+, glib
+, openssl
+, sqlite
+, capnproto
+, ensureNewerSourcesForZipFilesHook
+, pythonSupport ? true
+, pythonPackages ? null
 }:
 
 assert pythonSupport -> pythonPackages != null;
 
 rustPlatform.buildRustPackage rec {
   pname = "sequoia";
-  version = "0.18.0";
+  version = "0.19.0";
 
-  src = fetchFromGitLab {
-    owner = "sequoia-pgp";
-    repo = pname;
+  src = fetchgit {
+    url = "https://gitlab.com/sequoia-pgp/sequoia.git";
     rev = "v${version}";
-    sha256 = "18acv0185y51yz6jwchi1vf701shz37z5qmnzpq6z419lpjdaskd";
+    sha256 = "10ag8ry5bqbh9r0m5zj2rjw4z5288x6101ky3plvs9xyq5ahfwq1";
+    # We wouldn't be able to have this with fetchFromGitLab
```

Comment:
Why do we want this? I'm not sure I understand comment you replaced below, but maybe something like `Keep .git because it is needed for openpgp-ffi checks`?

Reply:
Correct. Previously there was a part of the `postPatch` that fixed some `git grep` commands in the original Makefiles. But I noticed there were more of these and I figured it'd be easier to `leaveDotGit` and have less such not absolutely necessary parts of the `patchPhase`.

Reply:
I explained it more thoroughly in a comment.

Reply:
Thanks :+1: